pytils/test/test_numeral.py

```diff
@@ -67,7 +67,7 @@ def testChoosePluralVariantsInStr(self):
             "гвоздь"
         )
         self.assertEqual(
-            pytils.numeral.choose_plural(5, "гвоздь, гвоздя, гвоздей\, шпунтов"),
+            pytils.numeral.choose_plural(5, r"гвоздь, гвоздя, гвоздей\, шпунтов"),
```

Comment:
```suggestion
            pytils.numeral.choose_plural(5, "гвоздь, гвоздя, гвоздей, шпунтов"),
```

why do we need `r` here? Maybe we're supposed to remove unneeded `\`?

Reply:
Oh, it's testing escaped variant. Nevermind.